test/Microsoft.NET.Publish.Tests/GivenThatWeWantToPreserveCompilationContext.cs

```diff
@@ -35,6 +35,7 @@ public void It_publishes_the_project_with_a_refs_folder_and_correct_deps_file()
                     propertyGroup.Should().NotBeNull();
 
                     propertyGroup.Add(new XElement(ns + "DisableImplicitFrameworkReferences", "true"));
+                    propertyGroup.Add(new XElement(ns + "DisableLockFileFrameworks", "true"));
```

Comment:
I've filed https://github.com/dotnet/sdk/issues/536 to investigate when the MSBuild issue is fixed